docs/source/tutorials/pytorch_encoder.rst

```diff
@@ -255,3 +251,225 @@ Current results are as follows, on a nVidia V100 (PyTorch 1.9, Triton 1.1, xForm
     | --------- | ----------------- | ------------------ | ------------------ |
     | xformers  | 89                | 1182               | 2709               |
     | pytorch   | 155               | 1950               | 4117               |
+
+
+
+Build an `xFormer` model with Hydra
+-----------------------------------
+
+Alternatively, you can use [Hydra](https://hydra.cc/) to build an xFormer model. 
+We've included an example [here](https://github.com/facebookresearch/xformers/tree/main/examples/build_model).
+The example replicates the model from the above example and demonstrates one way to use Hydra to minimize config duplication. 
+
+
+
+.. code-block:: yaml
+
+    defaults:
+        - /stack@xformer.stack_configs: 
```

Comment:
Would it be out of scope to add some more information as to what these do in this doc page for people who are unfamiliar? This is a pretty advanced usage of Hydra which may be alright for users of Hydra, but may require some onboarding.

Or maybe I'm completely wrong cc @omry

Reply:
thanks for taking a look @SeanNaren  - yea, you are right, this is "advanced" Hydra :) I've put more context in the example config files. I can certainly add a bit more context in the doc.